string robot_name

float64 robot_x
float64 robot_y
float64 robot_theta
float64 robot_velocity
float64 robot_time
float64 robot_xf
float64 robot_yf
float64 robot_state
float64 robot_xb
float64 robot_yb

float32[] robot_plan_x
float32[] robot_plan_y
